src/header.php

```diff
@@ -80,13 +80,13 @@
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
 <?php
 if(! isset($META_TITLE)){
-	$META_TITLE = 'PvPoke | Open-Source Battle Simulator, Rankings &amp; Team Building for Pokemon GO PvP';
+	$META_TITLE = 'PvPoke | Open-Source Battle Simulator, Rankings &amp; Team Building for Pokémon GO PvP';
```

Comment:
For style/brand reasons, I'd like to keep the word "Pokemon" without the accent throughout the site.